esmf/meta.yaml

```diff
@@ -0,0 +1,26 @@
+package:
+  name: esmf
+  version: 6.3.0rp1
+
+build: 
+  number: 0
+  skip: True # [osx]
+  skip: True # [win]
+  skip: True # [py3k]
```

Comment:
This line could be the cause of the skip. esmf doesn't depend on Python, so no need for this skip.